App/screens/photos.tsx

```diff
@@ -113,7 +113,8 @@ class Photos extends Component<Props> {
         />
       )
     } else {
-      const { photo, error } = row.item.processingPhoto
+      const { photo, state, error } = row.item.processingPhoto
+      const color = state === 'pending' ? 'white' : 'blue'
```

Comment:
Considering that `color` is another style attribute used to change the font color, why not just directly call this variable `backgroundColor`.

Reply:
Yea totally agree. This is temporary code that will be thrown out as soon as we make the real UI here, so wasn't worried too much about the nice details like that ;)